### PR TITLE
Messaging ui

### PR DESCRIPTION
**Pull request by chalynpai**
messagingUI pr

### CODE REVIEW

**Comment by echanlee** [comment]
I think it would look nice if the typing message box was within the messages box

File: src/Messages.js [hunk 1]
```diff
@@ -120,20 +130,41 @@ class Messages extends React.Component {
...
-            </div>
-          )):
-          <header>Start a conversation!</header>}
+            //Checks which user is messaging
+             names[index]==this.state.currentName ?
+              <div className = "UserOne">
+                <p1>{timeStamps[index]}</p1>
+                <p>{names[index]} : {msg}</p>
+              </div>
+            :
```
Comment (echanlee) [inline src/Messages.js]: can you reformat indentation to make it more clear which ternary statement the `:` belongs to

**Review by echanlee — CHANGES_REQUESTED** [review]

File: src/Messages.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	bottom: 5px;
+	width: 70%;
+	padding: 5px;
+	margin-left: 5px;
+	border-radius: 5px;
+	position: sticky;
+	border-color:#0081a7;
+}
+.MessageContainer button{
+    color: white;;
```
Comment (jj7lin) [inline src/Messages.css:59]: Maybe use a different color for the time text? It isn't as clear in the coral message box so it's harder to see

Comment (jj7lin) [reply src/Messages.css:59]: Can the button align to the right of the conversation container so they're better spaced out?

**Review by jj7lin — CHANGES_REQUESTED** [review]
some minor changes, but overall looks good

**Review by echanlee — APPROVED** [review]

**Review by jj7lin — APPROVED** [review]

**Pull request merged**